oaknational/Oak-Web-Application
# feat: lesson listing use new api

## Description

Music year: {owa_music_year}

- Lesson listing to use new api
- Update tests
- Update fixtures

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2353--oak-web-application.netlify.thenational.academy
Go to lesson listing pages, should be the same except legacy pages should now also have correct lesson meta data (foundation / year etc)

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
